velox/exec/Driver.cpp

```diff
@@ -340,22 +340,70 @@ void Driver::enqueueInternal() {
 
 // Call an Oprator method. record silenced throws, but not a query
 // terminating throw. Annotate exceptions with Operator info.
-#define CALL_OPERATOR(call, operator, methodName)                       \
+#define CALL_OPERATOR(call, operatorPtr, operatorId, operatorMethod)    \
   try {                                                                 \
     threadNumVeloxThrow() = 0;                                          \
+    opCallStatus_.start(operatorId, operatorMethod);                    \
     call;                                                               \
-    recordSilentThrows(*operator);                                      \
+    opCallStatus_.stop();                                               \
```

Comment:
So we are resetting the timer on each driver run loop, and we need to inspect the values while it is stuck, is it correct?  Curious what is the interface for doing this debugging, are we attaching debugger to the process or there is an API to fetch the stats on demand?

Reply:
@Yuhta 
The idea is to make it available in few different ways:
1. Already in this change: Driver::toString() and Driver::toJson(). Presto_cpp has an HTTP endpoint where you can fetch the info about the task - it will include the new fields.
2. Task runtime metric (can be seen in the Presto Query UI): Per task I plan to expose the longest operator in the shape of akin to "task/stage.stuck_operator.op_call" N ms. Need change in presto_cpp after this.
3. As ODS counter(s), so we can quickly identify the affected hosts.